app/components/ExecutionNewPageSummary.js

```diff
@@ -0,0 +1,302 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+import { withStyles } from '@material-ui/core/styles'
+
+
+import List from '@material-ui/core/List'
+import ListItem from '@material-ui/core/ListItem'
+import ListItemIcon from '@material-ui/core/ListItemIcon'
+import ListItemText from '@material-ui/core/ListItemText'
+
+import Typography from '@material-ui/core/Typography'
+import Avatar from '@material-ui/core/Avatar'
+import Grid from '@material-ui/core/Grid'
+import FormLabel from '@material-ui/core/FormLabel'
+import FormGroup from '@material-ui/core/FormGroup'
+
+
+import {
+  selectDatasets,
+} from '../reducers/dataset'
+
+import {
+  selectSchedulers,
+  selectCurrentScheduler,
+} from '../reducers/cpacpy'
+
+import {
+  ExecutionCurrentBackendIcon,
+  PipelineIcon,
+  PipelineStepIcon,
+  DatasetIcon,
+  DatasetSubjectIcon,
+  DatasetSiteIcon,
+  DatasetSessionIcon,
+  SchedulerIcon,
+  SchedulerParamIcon,
+} from '../components/icons'
+
+import cpac from '@internal/c-pac'
+import {fromJS} from "immutable";
+
+class ExecutionNewPageSummary extends Component {
```

Comment:
Wouldnt it be `ExecutionSummary` ? It is used in other places, besides the `new execution` page.

Reply:
The class name is now `ExecutionSummary`.